src/App.tsx

```diff
@@ -1,5 +1,46 @@
 import { IndexPage } from "./pages/IndexPage";
+import { Greeting } from "./components/Content/SelectedPage";
+import { createBrowserRouter, RouterProvider, Route } from "react-router-dom";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <IndexPage />,
+    children: [
+      {
+        path: "/",
+        element: (
+          <Greeting displayText="Welcome, please select a resource to continue!" />
+        ),
+      },
+      {
+        path: "/movies/",
+        element: <Greeting displayText="Movies" />,
+      },
+      {
+        path: "/planets/",
+        element: <Greeting displayText="Planets" />,
+      },
+      {
+        path: "/ships/",
+        element: <Greeting displayText="Ships" />,
+      },
+      {
+        path: "/species/",
+        element: <Greeting displayText="Species" />,
+      },
+      {
+        path: "/vehicles/",
+        element: <Greeting displayText="Vehicles" />,
+      },
+      {
+        path: "/characters/",
+        element: <Greeting displayText="Characters" />,
```

Comment:
for displayText, consider using values from the enum we setup, that way you get type safety and can change it only in one place if it needs to change

Reply:
Additionally, consider using `swapiResourceMetadata[SwapiResources.People].internalUrlPath` for the path value.